blueprints/codes/latex_formula.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class LatexFormula:
+    return_symbol: str
+    result: str
+    equation: str = ""
+    numeric_equation: str = ""
```

Comment:
add docstrings  (also mention that the units are included in the numeric equation )